docs/configuration.adoc

```diff
@@ -97,6 +97,7 @@ Each Component must be configured in corresponding configuration file according
             GAV = <MAVEN distribution artifacts> (can be configured using Expression Language)
             DEB = <DEBIAN distribution artifacts> (can be configured using Expression Language)
             RPM = <RPM distribution artifacts> (can be configured using Expression Language)
+            docker = <Docker distribution artifacts>
```

Comment:
"artifact".  Currently only one image per component is supported. pls add also example of the value ("test/test-component")

Reply:
Fixed